hub/api/tensor.py

```diff
@@ -120,10 +125,18 @@ def __iter__(self):
         for i in range(len(self)):
             yield self[i]
 
-    def numpy(self):
+    def numpy(self, aslist=False) -> Union[np.ndarray, List[np.ndarray]]:
         """Computes the contents of a tensor in numpy format.
 
+        Args:
```

Comment:
Please add the error that is raised in docstring.

Reply:
no error is being explicitly raised by this function

Reply:
i don't want us to get in the habit of copy pasting error raise strings up the call stack

Reply:
i added this because it's API facing